returnn/util/file_cache.py

```diff
@@ -53,11 +103,11 @@ class FileCache:
     def __init__(
         self,
         *,
-        cache_directory: str = "$TMPDIR/$USER/returnn/file_cache",
-        cleanup_files_always_older_than_days: float = 31.0,
-        cleanup_files_wanted_older_than_days: float = 7.0,
-        cleanup_disk_usage_wanted_free_ratio: float = 0.2,  # try to free at least 20% disk space
-        num_tries: int = 3,  # retry twice by default
+        cache_directory: str = DEFAULT_CACHE_DIR,
+        cleanup_files_always_older_than_days: float = DEFAULT_CLEANUP_FILES_ALWAYS_OLDER_THAN_DAYS,
+        cleanup_files_wanted_older_than_days: float = DEFAULT_CLEANUP_FILES_WANTED_OLDER_THAN_DAYS,
+        cleanup_disk_usage_wanted_free_ratio: float = DEFAULT_CLEANUP_DISK_USAGE_WANTED_FREE_RATIO,
+        num_tries: int = DEFAULT_NUM_TRIES,
```

Comment:
I don't see the benefit in rewriting the code like this. This just makes it unnecessarily more complicated. I would keep the old code.